doc/source/dev/testing.rst

```diff
@@ -4,27 +4,27 @@
 Testing in ODL
 ##############
 
-ODL is tested using pytest_ and has four main types of tests and verification measures that are supposed to test the code in different ways. 
-These are listed below along with the command to run them.
+ODL tests are run using pytest_, and there are several types:
+
 
 ==============  =========================  =======
 Name            Command                    Description
 ==============  =========================  =======
-Unit tests      `pytest`                   Test "micro-features" of the code, like testing that each parameter combination works, that the correct exceptions are raised, that the code works correctly in corner cases and with a wide range of input.
-Large-scale     `pytest --largescale`      Verify that the functionality works well in more complicated and realistic conditions with potentially large input data and longer running times.
-Doctests        `pytest`                   Run the code in the docstring examples and check the output against the documented ones. Mainly intended to validate the examples.
-Examples        `pytest --examples`        Run all examples in the `examples`_ folder. These are copy-paste friendly examples on how to use a function in a more complete context.
-Documentation   `pytest --documentation`   Run all examples in the documentation. Examples are occasionally used in the documentation to demonstrate a concept. Here we only check that the code is valid and actually runs.
+Unit tests      ``pytest``                 Test "micro-features" of the code
+Large-scale     ``pytest --largescale``    Unit tests with large inputs and more cases
+Doctests        ``pytest``                 Validate usage examples in docstrings
+Examples        ``pytest --examples``      Run all examples in the `examples`_ folder
```

Comment:
Is this link valid?

Reply:
Yes, it's just a standard hyperlink, the label is defined all the way down in the file.